src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -72,6 +72,16 @@
       login user
     end
 
+    context 'when asking for a non-existing login' do
+      before do
+        post :post_userinfo, params: { login: 'nonexistent', cmd: 'change_password', format: :xml }
```

Comment:
I don't understand this spec. What are you testing?

Reply:
I'm triggering an action that uses the `set_user` before_action with a non-existent user to asses that aborts it.